meta/FlexibleSUSY.m

```diff
@@ -500,6 +503,20 @@ FlexibleSUSY model file (FlexibleSUSY.m).
               ];
               FlexibleSUSY`FSCalculateDecays = False
            ];
+
+           With[{sarahVersion = DecomposeVersionString[SA`Version],
```

Comment:
It seems to me that the following `If` expression is a copy of the one in `CheckSARAHVersion[]`. Maybe both can be replaced by `VersionOrder[]` (or we move the If statement to a separate function)?

Reply:
Where should we put it? In `Utils`?

Reply:
Yes, this would be a good place.

Reply:
moved to #529